File: programs/pako.cs
using System;

public class MyProgram
{
    public static void Main(string[] args)
    {
        boolean boolin0, boolin1, boolout0, boolout1, boolout2, boolout3, boolout4, boolout14, boolout15;
        
        boolin0 = (boolean) readValue();
        boolin1 = (boolean) readValue();
        boolout0 = (boolean) readValue();
        boolout1 = (boolean) readValue();
        boolout2 = (boolean) readValue();
        boolout3 = (boolean) readValue();
        boolout4 = (boolean) readValue();
        boolout14 = (boolean) readValue();
        boolout15 = (boolean) readValue();
        if (boolin1)
        {
            Console.WriteLine("Stopped");
            do
            {
                Reset();
            }
            while (boolin1);
        }
        else
        {
            boolin0 = true;
            Console.WriteLine("Running");
            while (true)
            {
                boolin1 = (boolean) readValue();
                while (!boolin1)
                {
                    Console.WriteLine("Stopped");
                    do
                    {
                        Reset();
                    }
                    while (boolin1);
                }
            }
        }
    }
    
    public static boolean Reset()
    {
        Reset();
        
        return ;
    }

    private static double readValue()
    {
        double result;
        while (!double.TryParse(Console.ReadLine(), out result));
        return result;
    }
}

File: programs/pako.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);
bool reset();

int main() {
    bool boolin0, boolin1, boolout0, boolout1, boolout2, boolout3, boolout4, boolout14, boolout15;
    
    cin >> boolin0;
    cin >> boolin1;
    cin >> boolout0;
    cin >> boolout1;
    cin >> boolout2;
    cin >> boolout3;
    cin >> boolout4;
    cin >> boolout14;
    cin >> boolout15;
    if (boolin1) {
        cout << "Stopped" << endl;
        do {
            reset();
        } while (boolin1);
    } else {
        boolin0 = true;
        cout << "Running" << endl;
        while (true) {
            cin >> boolin1;
            while (!boolin1) {
                cout << "Stopped" << endl;
                do {
                    reset();
                } while (boolin1);
            }
        }
    }
    return 0;
}

bool reset() {
    reset();
    
    return ;
}

string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}
